diff --git a/collector/netdev_linux.go b/collector/netdev_linux.go
index 4ea09a3..04796af 100644
--- a/collector/netdev_linux.go
+++ b/collector/netdev_linux.go
@@ -32,7 +32,7 @@ var (
 )

 func getNetDevStats(ignore *regexp.Regexp) (map[string]map[string]string, error) {
-	file, err := os.Open(procFilePath("net/dev"))
+	file, err := os.Open(procFilePath("1/net/dev"))
 	if err != nil {
 		return nil, err
 	}
